libraries/TGRSIFormat/TChannel.cxx

```diff
@@ -401,8 +401,8 @@ double TChannel::CalibrateENG(double charge) {
    //integration parameter.
    if(ENGCoefficients.size()==0)
       return charge;
-   double cal_chg = 0.0;
-   for(int i=0;i<ENGCoefficients.size();i++){
```

Comment:
The way Vinzenz changed it to does the same thing, just a bit more
efficiently.  var = vector.at(0) is slightly faster than variable
=vector.at(0)*
charge^0.



On Mon, Jul 13, 2015 at 5:41 PM, Ryan Dunlop <notifications@github.com>
wrote:

> In libraries/TGRSIFormat/TChannel.cxx
> <https://github.com/GRIFFINCollaboration/GRSISort/pull/387#discussion_r34515499>
> :
>
> > @@ -401,8 +401,8 @@ double TChannel::CalibrateENG(double charge) {
> >     //integration parameter.
> >     if(ENGCoefficients.size()==0)
> >        return charge;
> > -   double cal_chg = 0.0;
> > -   for(int i=0;i<ENGCoefficients.size();i++){
>
> Why did we change this line? Was it causing problems?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GRIFFINCollaboration/GRSISort/pull/387/files#r34515499>
> .
>